### PR TITLE
Basic review module requirement complete

### PR DESCRIPTION
**Pull request by gocodezhang**
1. Added a middleware "logger" in the server - every time received request, it will console log the request method and url
2. Added jquery in package.json
3. Added back the eslint file

### CODE REVIEW

File: client/src/components/ratings-and-reviews/add-review-form/AddReviewForm.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          <p>
+            What is your nickname?
+            <small> (60 characters max)</small>
+          </p>
+          <textarea name="name" maxLength={60} placeholder="jackson11" rows={3} value={reviewObj.name} onChange={inputHandler}></textarea>
+          <small>For privacy reasons, do not use your full name or email address</small>
+        </div>
+        <div className="email input">
+          <p>
+            What is your nickname?
```
Comment (HeyMaximus) [inline client/src/components/ratings-and-reviews/add-review-form/AddReviewForm.jsx]: Do you mean to ask "What is your email" here?

**Review by HeyMaximus — APPROVED** [review]

File: client/src/components/ratings-and-reviews/add-review-form/AddReviewForm.jsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  function inputHandler(e, url) {
+    const characterArr = ['Size', 'Width', 'Comfort', 'Quality', 'Length', 'Fit'];
+    if (characterArr.includes(e.target.name)) {
+      const { id } = characteristics[e.target.name];
+      setReviewObj({
+        ...reviewObj,
+        characteristics: { ...reviewObj.characteristics, [id]: Number(e.target.value) },
+      });
+    } else if (e.target.name === 'photos') {
+      setReviewObj({ ...reviewObj, photos: reviewObj.photos.concat(url) });
```
Comment (HeyMaximus) [inline client/src/components/ratings-and-reviews/add-review-form/AddReviewForm.jsx:42]: Does concat work here? The url parameter (reader.result) is presented here in the form of a string from readAndPreview()

**Review by HeyMaximus — APPROVED** [review]

File: client/src/components/ratings-and-reviews/add-review-form/AddReviewForm.jsx (hunk 2 above)
Comment (HeyMaximus) [reply client/src/components/ratings-and-reviews/add-review-form/AddReviewForm.jsx:42]: Nevermind! I mistakenly thought concat only takes array. We good.

File: client/src/components/ratings-and-reviews/add-review-form/AddReviewForm.jsx (hunk 1 above)
Comment (gocodezhang) [reply client/src/components/ratings-and-reviews/add-review-form/AddReviewForm.jsx]: yes, thanks for catching that

**Pull request closed without merging**